docs/reference/services/app.rst

```diff
@@ -0,0 +1,25 @@
+.. _AppService:
+
+.. py:currentmodule:: craft_application.services.base
+
+``AppService``
+==============
+
+The ``AppService`` is the base class from which all services must inherit.
```

Comment:
The `ProjectService` doc discusses its higher-level purpose, scope, and operation. Why do we describe the `AppService` here in terms of its form and perspective as a class?